VideoPlusChat/src/main/java/com/example/videochatcomposesample/MainActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.videochatcomposesample
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import com.example.videochatcomposesample.ui.theme.VideoChatComposeSampleTheme
+import com.example.videochatcomposesample.ui.views.MainPage
+import com.google.android.exoplayer2.ExoPlayer
+import com.google.android.exoplayer2.MediaItem
+import com.microsoft.device.dualscreen.windowstate.WindowState
+import com.microsoft.device.dualscreen.windowstate.rememberWindowState
+
+// TODO Animate Using Constraint Layouts
+
+/*
+Levi O. Notes:
+Currently the keyboard toggle is not the best when it comes to the vertical fold dual alignment (it is meant to go from video on top and chat on bottom
+to a single pane row with chat and video alongside and keyboard on bottom).
+I have tried to work with shifting focuses as you can see a small example in the MainPage.kt at the bottom, but this does not work, at least with how
+I implemented it. I believe a better way to do this is to embed the app's composable into views and use constraint layouts and a global keyboard listener,
+however I believe this would take away from the idea of creating a pure compose application and thus I have not done it.
+ */
+
+class MainActivity : ComponentActivity() {
+
+    private lateinit var player: ExoPlayer
+    private lateinit var windowState: WindowState
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        player = ExoPlayer.Builder(this).build()
+        setContent {
+            windowState = rememberWindowState()
+            VideoChatComposeSampleTheme {
+                Box(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .background(MaterialTheme.colors.background)
+                ) {
```

Comment:
In general, I would not recommend wrapping composables in a Box unless you need it for specific alignment/overlapping purposes, because this helps make the code more readable and easier to change in the future. 

I think you are using a few boxes throughout your composables for two main reasons:
1. To provide a background color
2. To apply modifier attributes to an entire composable

For case 1, you could replace the Box with a [Surface](https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#Surface(androidx.compose.ui.Modifier,androidx.compose.ui.graphics.Shape,androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Color,androidx.compose.foundation.BorderStroke,androidx.compose.ui.unit.Dp,kotlin.Function0)), or you could also just use the [background Modifier attribute](https://developer.android.com/reference/kotlin/androidx/compose/foundation/package-summary#(androidx.compose.ui.Modifier).background(androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Shape)).

For case 2, it's better to just directly add the modifier attributes to the composable you're working with.

I will also comment on the specific lines of code where I think you could remove boxes!